ffig/templates/ffig.macros

```diff
@@ -16,7 +18,7 @@
 {%- macro comma_separated_list(items, leading_comma=False, trailing_comma=False) -%}
     {%- for item in items -%}
         {%- if leading_comma and loop.first %}, {% endif -%}
-        {{ caller(item) }}
+            {{ caller(item, *varargs) }}
```

Comment:
to clarify, this expands so that _each_ argument is followed by the extra args given to the function.

Reply:
one would get: `caller(a, extra_args), caller(b, extra_args), caller(c, extra_args)...`

Reply:
Yeah. It's up to `caller` (the macro that called into this utility macro) to do something (or not) with the extra parameters.

They weren't intended to be extra list items; just extra data for the callback to possibly use.